java/src/jmri/jmrix/direct/Throttle.java

```diff
@@ -34,6 +36,11 @@ public Throttle(DccLocoAddress address, CommandStation tc) {
         this.isForward = true;
     }
 
+    @Deprecated (since="5.13.3", forRemoval=true)
+    public Throttle(DccLocoAddress address, CommandStation tc) {
+        this(address, tc, jmri.InstanceManager.getDefault(SystemConnectionMemo.class));
+    }
```

Comment:
@bobjacobsen 
Is this deprecation needed? Or could the constructor be removed directly? I have hard to see that a script uses this constructor directly, or that some external program that's using JMRI as a library would use this constructor directly.